R/uvhydrograph-render.R

```diff
@@ -109,11 +109,24 @@ createPrimaryPlot <- function(data, month, useDownsampled=FALSE){
         xlab = paste("UV Series:", primaryInfo$date_lbl)
       )
 
-      #Don't add the right-side axis if we aren't actually plotting anything onto it
-      if((referenceExist && referenceSide == 4) || (comparisonExist && comparisonSide == 4)){
-        plot_object <- lines(plot_object, x=0, y=0, side = 4, reverse = primaryInfo$isInverted) %>%
-        axis(side = 4, las = 0)
-      }
+    #Don't add the right-side axis if we aren't actually plotting anything onto it
+    if((referenceExist && referenceSide == 4) || (comparisonExist && comparisonSide == 4)){
+      plot_object <- lines(plot_object, x=0, y=0, side = 4, reverse = primaryInfo$isInverted) %>%
+      axis(side = 4, las = 0)
+    }
+    
+    # still need gsplot to handle side 1 vs side 2 logging. See issue #414
+    # once that is working, use view to log the axes when appropriate
+    # could be added using logic in if statement above
+    # if(isLogged(data, ylimPrimaryData, primaryInfo$primarySeriesName)){
+    #   plot_object <- view(plot_object, side=primarySide, log='y')
+    # }
+    # if(comparisonExist && isLogged(data, ylimComparisonData, primaryInfo$comparisonSeriesName)){
+    #   plot_object <- view(plot_object, side=comparisonSide, log='y')
+    # }
+    # if(referenceExist && isLogged(data, ylimReferenceData, primaryInfo$referenceSeriesName)){
+    #   plot_object <- view(plot_object, side=referenceSide, log='y')
+    # }
```

Comment:
this is commented out because if one vertical side is logged, it logs all of them. This will hopefully be fixed in [gsplot #414](https://github.com/USGS-R/gsplot/issues/414)